### PR TITLE
Feature: People page bug fixes

### PR DESCRIPTION
**Pull request by bspeare**
# How to test

- `yarn workspace uids_base gulp --development`
- `blt ds --site=anthropology.uiowa.edu`
- `blt ds --site=journalism.uiowa.edu`
- Go to https://anthropology.local.drupal.uiowa.edu/people
- Verify that Form controls appear inline for safari at desktop breakpoint
- Verify that table row striping is gone
- Go to https://journalism.uiowa.edu/people and verify that page overflow is gone

### CODE REVIEW

**Comment by mark-bennett-uiowa** [comment]
Pulled down for testing.

**Comment by mark-bennett-uiowa** [comment]
Page overflow is FIXED on Journalism.

Table striping appears to be gone now on both Anthropology and Journalism sites:
Screen Shot 2021-05-20 at 11 28 43 AM


Form controls appear inline without overflow in Safari in my local instance:
Screen Shot 2021-05-20 at 11 30 48 AM

**Review by mark-bennett-uiowa — APPROVED** [review]
Tested and approved. No issues.

File: docroot/themes/custom/uids_base/scss/views/view-display-id-people.scss [hunk 1]
```diff
@@ -2,11 +2,20 @@
 @import "uids/assets/scss/_utilities.scss";
 
 .view-id-people {
+  .views-exposed-form {
+    .form--inline {
+      @include breakpoint(sm) {
+        flex-wrap: unset;
+      }
+    }
+  }
```
Comment (pyrello) [inline docroot/themes/custom/uids_base/scss/views/view-display-id-people.scss]: Is this something that should be applied more broadly than just to this one view?

**Pull request merged**